db/migrate/20180209173522_add_column_review_to_book.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnReviewToBook < ActiveRecord::Migration[5.1]
+  def change
+    add_column :books, :review, :string
+  end
```

Comment:
I would have expected reviews to be it's own table. It should act as a join table between books and users.